docs/opcodes/16.mdx

```diff
@@ -3,7 +3,7 @@ fork: Frontier
 group: Comparison & Bitwise Logic Operations
 ---
 
-*Index 1 is top of the stack. See [PUSH](/#60).*
+_Index 1 is top of the stack. See [PUSH](/#60)._
```

Comment:
This is repeated all over the place. Potentially can be extract?

Reply:
Yes, that’s possible